tasks/install_powershell.ps1

```diff
@@ -105,8 +105,19 @@ $date_time_stamp = (Get-Date -format s) -replace ':', '-'
 $msi_dest = Join-Path ([System.IO.Path]::GetTempPath()) "puppet-agent-$arch.msi"
 $install_log = Join-Path ([System.IO.Path]::GetTempPath()) "$date_time_stamp-puppet-install.log"
 
+ # If TLS 1.2 is not configured for the current session add it to the list of defined Security Protocols
```

Comment:
The commit message includes great context about why TLS 1.2 might not be configured and why it's necessary. Can you include a short version of that in this comment (ie. "old windows version, modern webserver")?

Reply:
Sure. I'll sort that out shortly.

Reply:
Updated!